odc/stats/plugins/gm.py

```diff
@@ -22,8 +22,9 @@ def __init__(
         self,
         bands: Tuple[str, ...],
         mask_band: str,
+        contiguity_band: Optional[str] = None,
         nodata_classes: Optional[Tuple[str, ...]] = None,
-        cloud_filters: Dict[str, Iterable[Tuple[str, int]]] = None,
+        cloud_filters: Dict[Tuple[str, ...], Iterable[Tuple[str, int]]] = None,
```

Comment:
```suggestion
        cloud_filters: Dict[Union[str, Tuple[str, ...]], Iterable[Tuple[str, int]]] = None,
```

from code below it looks like keys could be either a single string or a tuple of any number of strings.